mlir/test/e2e/conv2d_wrw_perf_config.cfg

```diff
@@ -0,0 +1,2 @@
+if not config.arch_support_mfma:
+  config.unsupported = True
```

Comment:
Nit: missing final newline (if you're in VS code, there might be a setting you'll need to enable to get this automatically, IIRC)

Reply:
Why do we want a newline? Is it coding guide or something?

Reply:
Yeah, files ending in a `\n` is a pretty standard bit of coding style, to the point most editors are or can be made to enforce it

The main reason is that, without one, `cat`ing your file will mess up the shell prompt by leaving it on the same line as the end of the file

But also, not having `\n` on the last line can confuse various scripts